src/modules/Reports/index.tsx

```diff
@@ -0,0 +1,5 @@
+import { Overdue } from './components/Overduetime';
+
+export const TicketAsTime = () => {
```

Comment:
Why to create extra component if there are no extra changes done here? We can just use the Overdue component directly wherever required